ecosystem/resources/members/qBraid.toml

```
@@ -0,0 +1,17 @@
+name = "qBraid"
+url = "https://github.com/qBraid/qBraid"
+description = "The qBraid-SDK is a Python toolkit for cross-framework abstraction, transpilation, and execution of quantum programs."
+licence = "GNU General Public License (GPL)"
+contact_info = "ryanhill@qbraid.com"
+alternatives = "There are some alternatives to individual modules within the qBraid-SDK e.g. to execute a Qiskit circuit on an AWS device, you could use the qiskit-braket-provider. However, there aren't any other existing projects that support the same matrix of quantum program types and backends through a single, unified interface."
+affiliations = "qBraid Co."
+website = "_No response_"
```

```suggestion
website = "_No response_"
```

I think this will cause a broken link on qiskit.org...

---

I can't approve your suggestion, I don't have the right to push onto repo branch